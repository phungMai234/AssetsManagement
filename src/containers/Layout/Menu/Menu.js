import React from 'react';

import Wrapper from './Menu.styles';

const Menu = () => {
  return <Wrapper className="menu">menu</Wrapper>;
};

export default Menu;
