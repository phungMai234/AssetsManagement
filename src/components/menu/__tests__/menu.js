import React from 'react';

import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import Menu from '../menu';

describe('Menu', () => {
  it('should render correctly', () => {
    const container = render(
      <Router>
        <Menu />
      </Router>,
    );
    expect(container.container).toMatchSnapshot();
  });
});
