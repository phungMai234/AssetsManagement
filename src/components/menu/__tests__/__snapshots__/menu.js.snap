// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render correctly 1`] = `
<div>
  <ul
    class="nav flex-column nav-pills nav-fill nav-menu"
  >
    <li
      class="nav-item"
    >
      <a
        class="nav-link "
        href="/dashboard/categories"
      >
        <svg
          fill="none"
          height="20"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"
          />
          <polyline
            points="9 22 9 12 15 12 15 22"
          />
        </svg>
        <span>
          Quản lý các loại tài sản
        </span>
      </a>
    </li>
    <li
      class="nav-item"
    >
      <a
        class="nav-link "
        href="/dashboard/assets"
      >
        <svg
          fill="none"
          height="20"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <ellipse
            cx="12"
            cy="5"
            rx="9"
            ry="3"
          />
          <path
            d="M21 12c0 1.66-4 3-9 3s-9-1.34-9-3"
          />
          <path
            d="M3 5v14c0 1.66 4 3 9 3s9-1.34 9-3V5"
          />
        </svg>
        <span>
          Quản lý tài sản
        </span>
      </a>
    </li>
    <li
      class="nav-item"
    >
      <a
        class="nav-link "
        href="/dashboard/delivery_reports"
      >
        <svg
          fill="none"
          height="20"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M14 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V8z"
          />
          <polyline
            points="14 2 14 8 20 8"
          />
          <line
            x1="16"
            x2="8"
            y1="13"
            y2="13"
          />
          <line
            x1="16"
            x2="8"
            y1="17"
            y2="17"
          />
          <polyline
            points="10 9 9 9 8 9"
          />
        </svg>
        <span>
          Quản lý biên bản bàn giao
        </span>
      </a>
    </li>
  </ul>
</div>
`;

exports[`Menu should render correctly with tab categories 1`] = `<div />`;
